autoflake.py

```diff
@@ -1381,6 +1395,12 @@ def _main(argv, standard_out, standard_error, standard_input=None) -> int:
         )
         return 1
 
+    if args.check and args.check_diff:
```

Comment:
You can do this with a mutually exclusive group: https://docs.python.org/3/library/argparse.html#mutual-exclusion

Reply:
Very nice, TIL. Done.